src/Npgsql/NpgsqlTypes/NpgsqlTsQuery.cs

```diff
@@ -253,18 +253,27 @@ public static NpgsqlTsQuery Parse(string value)
         if (pos >= value.Length)
             goto Finish;
         ch = value[pos];
-        if (ch == '*')
+        switch (ch)
```

Comment:
Is this hot enough to 
```suggestion
        switch (ch | 0x20)
```
so to have the `case`s only for lowercase chars, thus saving some branches and code-size?
See [sharplab](https://sharplab.io/#v2:EYLgxg9gTgpgtADwGwBYA+ABATARgLABQGAzAATakDCpA3oaQ+WWABYCGUpAsjgBSsdSrAJSl6jOgUbTSAZwDuASwAurUvxbDxM2tp2MwbWTFIByAFSmQ5AOxnLAbj36hRk6banS0MwEErtn6mTlIuBm5mwF4+pgBCARh2ccHO+obGZmDRnKaUCUl5IWHhGaYAJtlmACL51SmhxWUwAGZsAK4ANsrWiULsUAB0XIoAdgBqbB1tMEX6AL56Cw1iyyR9glxYGoIiK9KSLgoqahqkaKQADAhYF1rL+6k66e6WPUmOjzLPZp5vQbPFb6mKJ/ZIAsJArKgwqfaRAiqgmrglxNVqdbqBASDYbjSbTZGMJbSJZzIA==)

Reply:
I definitely don't think this code is hot enough for micro-optimization.. In fact, we should consider removing the parsing logic altogether, as this PR does for the geometry types.